### PR TITLE
Add Chrome extension sample

### PR DESCRIPTION
**Pull request by ImJasonH**

### CODE REVIEW

**Comment by ImJasonH** [comment]
I'm a Google employee, my @google.com email address wasn't my GitHub profile's primary email address, but it is now.

File: chrome-extension/background.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
+// http makes an HTTP request and calls callback with parsed JSON.
```
Comment (tswast) [inline chrome-extension/background.js]: Missing Apache 2 Google license header.

File: chrome-extension/package.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#! /bin/bash
```
Comment (tswast) [inline chrome-extension/package.sh]: small script, but should have license header, too.

**Comment by ImJasonH** [comment]
Silly CLA bot, I had reset my @gmail.com account to primary, my bad.

**Comment by tswast** [comment]
LGTM.

Thanks!

**Pull request merged**